Purchases/Purchasing/ProductDetails.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  Product.swift
+//  Product
+//
+//  Created by Andrés Boedo on 7/16/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
```

Comment:
//
//  Copyright RevenueCat Inc. All Rights Reserved.
//
//  Licensed under the MIT License (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//      https://opensource.org/licenses/MIT
//
//  Product.swift
//
//  Created by Andrés Boedo on 7/16/21.
//